tests/integration/src/testing/testnet.rs

```diff
@@ -432,20 +422,13 @@ async fn create_account(
 
 /// Build a note
 async fn create_note(client: &mut Client, scenario: &Scenario, params: &CreateNoteParams) -> Note {
-    let note_program = params.note.unwrap_program();
-    let note_script = miden_client::note::NoteScript::from_parts(
-        note_program.mast_forest().clone(),
-        note_program.entrypoint(),
-    );
+    use miden_client::note::WellKnownNote;
+    let note_script = WellKnownNote::P2ID.script();
```

Comment:
This is just always going to create the P2ID note, rather than a note provided via `CreateNoteParams` - I would expect that the caller would invoke `create_note` with a specific concrete note they want to create, otherwise this function should just be called `create_p2id_note`, but that sort of defeats the point of having a helper function like this.